config/main.go

```diff
@@ -93,7 +93,9 @@ func initialize() {
 	options.SetDefault(Keys.CertsFromEnv, false)
 	options.SetDefault(Keys.Port, "3000")
 	options.SetDefault(Keys.SubsHost, "https://subscription.api.redhat.com")
-	options.SetDefault(Keys.CaPath, "./resources/ca.crt")
+	options.SetDefault(Keys.CaPath, "../resources/ca.crt")
+	options.SetDefault(Keys.Cert, "../test_data/test.cert")
+	options.SetDefault(Keys.Key, "../test_data/test.key")
```

Comment:
i would add a comment here stating that the defaults here are really intended for our test cases

Reply:
done!